core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -10532,4 +10532,10 @@ private void checkLoptOptimizeJoinRule(LoptOptimizeJoinRule rule) {
         .withRule(CoreRules.JOIN_CONDITION_PUSH, CoreRules.FILTER_INTO_JOIN)
         .check();
   }
+
+  @Test void testAggregateMinMaxToLimitRule() {
```

Comment:
we should take advantage of the new Quidem tests with optimization rules to check this

Reply:
There are some problems when executing Quidem test, and the error is as follows:
```
> Caused by: java.util.NoSuchElementException
> 	at java.util.AbstractList$Itr.next(AbstractList.java:364)
> 	at org.apache.calcite.rel.core.Sort.explainTerms(Sort.java:267)
> 	at org.apache.calcite.rel.AbstractRelNode.getDigestItems(AbstractRelNode.java:414)
> 	at org.apache.calcite.rel.AbstractRelNode.deepHashCode(AbstractRelNode.java:397)
```
Maybe it has something to do with this
```
  public RexNode desc(RexNode node) {
    return call(SqlStdOperatorTable.DESC, node);
  }
```
I'll try to solve this problem first.

Reply:
Sorry, ignore the error above. It's due to some other reasons, not related to this PR. I'll check what the problem is later. The test has been added, please review it again, thanks!